packages/@glimmer/reference/test/template-test.ts

```diff
@@ -62,6 +62,51 @@ module('@glimmer/reference: template', () => {
 
       ref.get('foo');
     });
+
+    test('multiple access', assert => {
+      let pullCount = 0;
+      let setCount = 0;
+
+      let component = {
+        get foo() {
+          pullCount++;
+          return pullCount;
+        },
+      };
+      let ref = new ComponentRootReference(
+        component,
+        new (class extends TestEnv {
+          setTemplatePathDebugContext(_ref: TemplatePathReference, key: string) {
```

Comment:
this API is only used in DEBUG, can we wrap this portion of the test in the DEBUG flag, or split it out into a separate test that is?

Reply:
@pzuraq is the idea to prep these tests for potentially running in prod?

Reply:
Ya, I think in general we’d like to be able to (not sure if there are other tests that break in prod atm, but Ember does run its tests in prod builds)